3.10/release-notes-api-changes310.md

```diff
@@ -37,13 +37,22 @@ The following HTTP APIs are affected:
 
 #### Cursor API
 
-The cursor API can now return two additional statistics values in its `stats` subattribute:
+The cursor API can now return additional statistics values in its `stats` subattribute:
 
 * *cursorsCreated*: the total number of cursor objects created during query execution. Cursor
   objects are created for index lookups.
 * *cursorsRearmed*: the total number of times an existing cursor object was repurposed. 
   Repurposing an existing cursor object is normally more efficient compared to destroying an 
   existing cursor object and creating a new one from scratch.
+* *cacheHits*: the total number of index entries read from in-memory caches for indexes
+  of type edge or persistent. This value will only be non-zero when reading from indexes
+  that have an in-memory cache enabled, and when the query allows using the in-memory
+  cache (i.e. using equality lookups on all index attributes).
+* *cacheMisses*: the total number of cache read attempts for index entries that could not
+  be served from in-memory caches for indexes of type edge or persistent. This value will 
+  only be non-zero when reading from indexes that have an in-memory cache enabled, the 
+  query allows using the in-memory cache (i.e. using equality lookups on all index attributes)
+  and the looked up values are not present in the cache.
```

Comment:
None of these stats appear to be a documented in Rest DocuBlocks 😢

Reply:
I think they need to be added to Documentation/DocuBlocks/Rest/Cursors/post_api_cursor.md.
I don't know how to add the new (and old) attributes in a structured way to the REST docu blocks. 
@Simran-B : do you know how to do this?

Reply:
Yes, will look into documenting them. DocuBlocks documentation is here BTW: https://github.com/arangodb/docs/blob/main/README.md#adding-a-new-docublock